### PR TITLE
Tutorials

### PR DESCRIPTION
**Pull request by danjust**
Edits to conda environment and requirements

### CODE REVIEW

File: requirements/common.txt [hunk 1]
```diff
@@ -1,3 +1,4 @@
 psycopg2
 SQLAlchemy
 numpy
+sparqlwrapper
```
Comment (Dan-Staff) [inline requirements/common.txt]: I don't think this should be here.
common.txt should only contain the requirements for the package, not the tutorial.
We shouldn't introduce a dependency on sparqlwrapper if it's not needed to use the package.

**Comment by Dan-Staff** [comment]
I've changed this to a merge request into documentation since we're already updating the tutorials there @danjust

**Comment by Dan-Staff** [comment]
OK, @danjust, I've updated the environment.yml to include sparqlwrapper and the other pip dependences not covered in the yml file (SQLAlchemy and psycopg2). Can you test it for me?

**Comment by danjust** [comment]
Works fine now @Dan-Staff

**Pull request merged**